docs/geckoview-quick-start.md

```diff
@@ -1,7 +1,9 @@
 ---
 layout: default
 title: Get Started with GeckoView
-nav_order: 2
+nav_order: 2 
+summary: How to use GeckoView in your Android app.
+tags: [GeckoView,Gecko,mozilla,android,WebView,mobile,mozilla-central,setup,quick start]
```

Comment:
FYI Google says it doesn't look at tags at all. This document is helpful: https://support.google.com/webmasters/answer/79812?hl=en

do you know how does the `summary` show up in the html? is it `<meta name="summary">`?

Reply:
I've also updated the head HTML template to add these meta tags. `summary` is displayed as a `description` meta tag.

Reply:
sounds good!